flask_admin/form/__init__.py

```diff
@@ -40,7 +40,7 @@ def recreate_field(unbound):
     return unbound.field_class(*unbound.args, **unbound.kwargs)
 
 
-if int(wtforms_version[0]) > 1:
+if int(float(wtforms_version[0])) > 1:
```

Comment:
This is strange. What's the type of wtforms_version that it needs `float` now? Also, don't need `int` conversion - `float` would work as expected.